### ISSUE TITLE
Remove release_urlbase as an option in iojs+release

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/build#393]
It is my impression that the `release_urlbase` is deterministic. That is to say that whatever `disttype` you select will ultimately decide which `release_urlbase` the build will expect to find the headers.

I would like propose removing this option from CI and inferring the url from the selected disttype. This will get rid of the nasty case where we forget to set it correctly.

**Comment by jbergstroem** [comment]
I'd prefer if @rvagg chipped in here seeing how I haven't done much work in this area.

**Comment by rvagg** [comment]
I can't think of a good reason not to collapse it now. It was put there because at the time I was attempting to keep as much logic out of the Jenkins scripts as possible so we don't build up too much reliance on this beast, but that day has well passed now. Originally IIRC this was freeform text so it could point to anywhere. We've narrowed down the use of it a lot more now. Even the `custom` -> `rc` thing could be scripted away, the `custom` was a compromise I had to make when getting approval from others to land the code in `Makefile` that controls this.
